data/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.mapping

```diff
@@ -10,9 +10,15 @@ CLASS net/minecraft/client/gui/screens/worldselection/WorldSelectionList
 		ARG 9 worldSelectionList
 	METHOD moveSelection (Lnet/minecraft/client/gui/components/AbstractSelectionList$SelectionDirection;)V
 		ARG 1 ordering
+	METHOD refreshList (Ljava/util/function/Supplier;Z)V
+		ARG 1 supplier
```

Comment:
A more descriptive name would be something like `searchStringSupplier` or `filterStringSupplier`, as the supplied string is used to filter out any levels which do not contain the string as a substring of their level name or level ID.